velox/type/Type.h

```diff
@@ -392,13 +392,23 @@ using TypePtr = std::shared_ptr<const Type>;
 /// A struct which represents the parameters for a BigintEnumType.
```

Comment:
> A struct which represents

Drop this prefix as it is redundant. Start with an active verb. "Represents parameters for ..."